numpy/core/src/common/simd/avx512/reorder.h

```diff
@@ -214,13 +214,13 @@ NPY_FINLINE npyv_u16 npyv_rev64_u16(npyv_u16 a)
 
 NPY_FINLINE npyv_u32 npyv_rev64_u32(npyv_u32 a)
```

Comment:
Was this (and `npyv_rev64_f32`) ever called?

Reply:
I don't know, I'm just a translator ;-)